views/clucks/show.ejs

```diff
@@ -0,0 +1,15 @@
+<%- include('../partials/header') %>
+
+<div class="container-fluid">
+  <img class="img-fluid" src="<%= cluck.image_url %>" />
+
+  <div class="container-body">
```

Comment:
This part seems very similar to https://github.com/LanaShuvaeva/clucks/pull/1/files#diff-06f5bc32998b78c20ca546f3c3a87fe3R44

Maybe to avoid code duplication, we can move it into a partial?